src/main/java/com/intellij/rt/debugger/agent/CaptureStorage.java

```diff
@@ -23,98 +23,196 @@ protected Deque<CapturedStack> initialValue() {
     }
   };
 
+  private static final ThreadLocal<Boolean> THROWABLE_CAPTURE_DISABLED = new ThreadLocal<Boolean>() {
+    @Override
+    protected Boolean initialValue() {
+      return false;
+    }
+  };
+
   @SuppressWarnings("StaticNonFinalField")
   public static boolean DEBUG; // set from debugger
   private static boolean ENABLED = true;
 
+  private static final StackTraceElement ASYNC_STACK_ELEMENT =
+          new StackTraceElement("--- Async", "Stack.Trace --- ", "captured by debugger agent", -1);
```

Comment:
captured by Intellij IDEA debugger

Reply:
and probably remove the dot between stack and trace

Reply:
Dot is for styling as `printStackTrace` joins class name and method name with dot: "--- Async.Stack.Trace ---"

Reply:
`	at --- Async.Stack.Trace --- (captured by IntelliJ IDEA debugger)`